PythonScript/src/NotepadPlusWrapper.h

```diff
@@ -10,6 +10,7 @@
 #endif
 
 #include "GILManager.h"
+#include "Enums.h"

```

Comment:
Seems this include is not necessary and just needed in cpp file. Could you please check that.